system-test/jest.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "ODS SystemTests",
+  "globalSetup": "<rootDir>/src/setup.js",
+  "globalTeardown": "<rootDir>/src/teardown.js"
```

Comment:
I really like this approach using jest's `globalSetup` and `globalTeardown` hooks 🚀